src/Microsoft.FeatureManagement/Status.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// Describes how a feature's state will be evaluated.
+    /// </summary>
+    public enum Status
```

Comment:
Status seems to generic. I think it might need to be FeatureFlagStatus

Reply:
So just change this class name, but not the "Status" key when someone creates a feature definition? Or change both?

Reply:
Just the class name.